Source/Urho3D/IO/PackageFile.h

```diff
@@ -106,6 +113,11 @@ class URHO3D_API PackageFile : public Object
     /// Scan package for specified files.
     void Scan(ea::vector<ea::string>& result, const ea::string& pathName, const ea::string& filter, bool recursive) const;
 
+private:
+
+    /// Open the package file. Return true if successful.
+    bool OpenImpl(AbstractFile* file, unsigned startOffset = 0);
```

Comment:
This would be more useful if it was public. Then we would not need `MemoryBuffer` overload and any file could be used. Variant with string file name then is a convenience wrapper over this.